pandas/io/formats/excel.py

```diff
@@ -65,27 +65,156 @@ class CSSToExcelConverter:
         CSS processed by :meth:`__call__`.
     """
 
+    # source: matplotlib 3.3.3, matplotlib.colors.CSS4_COLORS
```

Comment:
can we just import these?

Reply:
possible of course, but is it worth introducing a dependency on mpl for excel manipulation?

Reply:
I used matplotlib nomenclature (_color_data.py), maybe something else will be more adequate